src/main/cpp/option_processor.cc

```diff
@@ -201,11 +201,22 @@ std::set<std::string> GetOldRcPaths(
         internal::FindRcAlongsideBinary(cwd, path_to_binary);
     candidate_bazelrc_paths = {workspace_rc, binary_rc, system_bazelrc_path};
   }
-  string user_bazelrc_path = internal::FindLegacyUserBazelrc(
-      SearchUnaryOption(startup_args, "--bazelrc", /* warn_if_dupe */ true),
-      workspace);
-  if (!user_bazelrc_path.empty()) {
-    candidate_bazelrc_paths.push_back(user_bazelrc_path);
+  vector<std::string> cmd_line_rc_files = SearchNaryOption(startup_args, "--bazelrc");
+  // Divide the cases where the vector is empty vs not, as `FindLegacyUserBazelrc` has a case
```

Comment:
nit: is this line too long? Please reflow if so

Reply:
I wasn't able to find any auto formatter provided in the repo, so manually adjusted it. You have any pointers, that'd be great for future reference.